### PR TITLE
[ R3-Corda-Ent ] Changes for using https for flux

### PR DESCRIPTION
**Pull request by srinivasansankaran**
Signed-off-by: srinivasan.sankaran <srinivasan.sankaran@accenture.com>

**Changelog**
 - Updated template,roles,molecule test and network yaml references to https from ssh for r3-corda-ent.
 - Updated the shared/setup readme file to reflect recent flux changes.
 - Updated shared/setup/flux role to use git username,password and url for https.
 - Removed references of ssh from setup/flux/role.
 - Updated the vars of the role from where shared/flux/setup role is called for r3-corda-ent.

 

**Reviewed by**
@suvajit-sarkar 
@jagpreetsinghsasan

 

**Linked issue**
#1220

### CODE REVIEW

File: platforms/shared/configuration/roles/setup/flux/tasks/main.yaml [hunk 1]
```diff
@@ -29,10 +23,10 @@
...
-    KUBECONFIG={{ kubeconfig_path }} helm upgrade --install flux-{{ network.env.type }}-helm-operator fluxcd/helm-operator --namespace default --set rbac.create=true --set git.timeout=200s --set git.pollInterval=2m --set git.url='{{ git_url }}' --set git.ssh.secretName=git-auth-{{ network.env.type }} --set-file git.ssh.known_hosts=flux_known_hosts --set helm.versions=v3 --kube-context="{{ kubecontext }}"
+    KUBECONFIG={{ kubeconfig_path }} helm upgrade --install flux-{{ network.env.type }} {{ playbook_dir }}/../../../platforms/shared/charts/flux --namespace default --set git.secretName=git-auth-{{ network.env.type }} --set rbac.create=true --set git.timeout=200s --set git.pollInterval=2m --set git.url='https://{{ git_username }}:{{ git_password }}@{{ git_repo }}' --set git.branch={{ git_branch }} --set git.label='sync-{{ network.env.type }}' --set git.path="{{ git_path }}"  --set registry.insecureHosts="{{ network.docker.url }}" --kube-context="{{ kubecontext }}"
```
Comment (suvajit-sarkar) [inline platforms/shared/configuration/roles/setup/flux/tasks/main.yaml:28]: Will this not break other DLT platforms for which don't have the https changes?

Comment (srinivasansankaran) [reply platforms/shared/configuration/roles/setup/flux/tasks/main.yaml:28]: @suvajit-sarkar Yes..It won't break other DLT platforms.

File: platforms/r3-corda-ent/configuration/samples/network-cordaent.yaml [hunk 2]
```diff
@@ -79,15 +79,14 @@ network:
       # Git Repo details which will be used by GitOps/Flux.
       # Do not check-in git_access_token
       gitops:
-        git_ssh: "ssh://git@github.com/<username>/blockchain-automation-framework.git"         # Gitops ssh url for flux value files 
+        git_https: "https://github.com/<username>/blockchain-automation-framework.git"         # Gitops https url for flux value files
```
Comment (sownak) [inline platforms/r3-corda-ent/configuration/samples/network-cordaent.yaml:82]: this should be merged with the git_push_url. There is no need to have two URLs pointing to the same path.

**Review by sownak — CHANGES_REQUESTED** [review]

File: platforms/r3-corda-ent/configuration/samples/network-cordaent.yaml (hunk 2 above)
Comment (srinivasansankaran) [reply platforms/r3-corda-ent/configuration/samples/network-cordaent.yaml:82]: @sownak  Okay.. Will remove git_push_url from network.yaml and keep only git_https

**Comment by srinivasansankaran** [comment]
Created new PR for the changes

**Pull request closed without merging**